packages/@okta/courage-for-signin-widget/scripts/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -x
+
+# Copy handlebars precompile babel plugin
+mkdir -p ../babel-plugin-handlebars-inline-precompile
+cp -rf ./node_modules/@okta/babel-plugin-handlebars-inline-precompile/* ../babel-plugin-handlebars-inline-precompile/
```

Comment:
does the webpack copy plugin support copy dir?

Reply:
I was running into an issue using webpack to do the copy. This babel plugin is used BY the webpack at a very early phase, so the plugin needs to be in place before we start compiling any source files. There may be a way to get it to work. If this is important, I can spend some more time on it.

Reply:
The problem is using wrapper build script is it doesn't support `webpack` parameters (although there is probably a quick and may not matter much for this package)
Would be good to avoid if possible (separate PR is fine).

Reply:
I see, you'd like to do something like `yarn build --watch` (instead of `npx webpack --watch`)?

Reply:
yeah and other parameters for `webpack` command.
it might not be big deal for this module but always trying to avoid build wrapper if possible.

Reply:
I moved this to webpack copy and removed the build script